src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/Dec2Hex.txt

```diff
@@ -0,0 +1,142 @@
+﻿﻿// ******** INTEGER PARAMETERS WITHOUT PLACES ********
+
+>> Dec2Hex(0)
+"0"
+
+>> Dec2Hex(Blank())
+"0"
+
+// Positive Integer Parameters
+>> Dec2Hex(165)
+"A5"
+
+>> Dec2Hex(3213)
+"C8D"
+
+>> Dec2Hex(549755813887)
+"7FFFFFFFFF"
+
+// Negative Integer Parameters
+>> Dec2Hex(-1)
+"FFFFFFFFFF"
+
+>> Dec2Hex(-165)
+"FFFFFFFF5B"
+
+>> Dec2Hex(-3213)
+"FFFFFFF373"
+
+>> Dec2Hex(-549755813888)
+"8000000000"
+
+// ******** DECIMAL PARAMETERS ********
+
+>> Dec2Hex(3.74)
+"3"
+
+>> Dec2Hex(-3.74)
+"FFFFFFFFFC"
+
+
+// ******** INTEGER PARAMETERS WITH PLACES ********
+
+>> Dec2Hex(45780)
+"B2D4"
+
+>> Dec2Hex(45780, 3)
+#Error(Kind=InvalidArgument)
+
+>> Dec2Hex(45780, 4)
+"B2D4"
+
+>> Dec2Hex(45780, 5)
+"0B2D4"
+
+>> Dec2Hex(45780, 6)
+"00B2D4"
+
+>> Dec2Hex(45780, -6)
```

Comment:
>> [](http://example.com/codeflow?start=0&length=1)

More tests with places == 0 and places == 10 (we should show that the edges still work)